R/stat-qq-line.R

```diff
@@ -0,0 +1,131 @@
+#' A quantile-quantile line
```

Comment:
I'd remove most of the stuff here (except for the new parameters), and instead write more in the `geom_qq` docs. I think it makes sense to document them together since they're typically used together